src/ProgressOnderwijsUtils/BenchTimer.cs

```diff
@@ -6,8 +6,8 @@ public sealed record BenchMark(TimeSpan Min, TimeSpan Max, TimeSpan Median, Time
 
     public static BenchMark Time(Action a, int numRuns)
     {
-        if (numRuns < 1) {
-            throw new ArgumentException("Need to test for at least 1 run", nameof(numRuns));
+        if (numRuns < 5) {
```

Comment:
of we doen de Outlier-removal op basis van %?  Dan is die #-runs check hier niet zo nodig.

of we maken het # outliers om te verwijderen ook een parameter?

Reply:
Ik heb er een param van gemaakt, plus wat api-docs om het uit te leggen.